src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.js

```diff
@@ -76,21 +68,54 @@ class CheckInQuestion extends Component {
   };
 
   render() {
-    const {styles, team: {tier}} = this.props;
+    const {isFacilitating, styles, team: {tier}} = this.props;
     const {editorState} = this.state;
     const canEdit = tierSupportsUpdateCheckInQuestion(tier);
     const isEditing = editorState.getSelection().getHasFocus();
+
+    const iconStyle = {
```

Comment:
for things that are static, it's nice to keep them out of the runtime loop. it's a little more performant, but more importantly, it makes it super obvious that it won't change

Reply:
Ok, I can move this.